src/ports/postgres/modules/deep_learning/madlib_keras_wrapper.py_in

```diff
@@ -440,16 +444,16 @@ def get_custom_functions_list(compile_params):
 
     """
     compile_dict = convert_string_of_args_to_dict(compile_params)
-    builtin_losses = dir(losses)
+    builtin_losses = update_builtin_losses(dir(losses))
     builtin_metrics = update_builtin_metrics(dir(metrics))
 
     custom_fn_list = []
     local_loss = compile_dict['loss'].lower() if 'loss' in compile_dict else None
-    local_metric = compile_dict['metrics'].lower()[2:-2] if 'metrics' in compile_dict else None
+    metric_list = ast.literal_eval(compile_dict['metrics'].lower()) if 'metrics' in compile_dict else []
```

Comment:
We call `ast.literal_eval()` as part of `validate_and_literal_eval_keys` which is called later in the flow, do we still need to add it here?

Reply:
Sounds good. Yes, I think we should add the similar try/catch for better error msg.